pkg/reconciler/ingress/resources/virtual_service.go

```diff
@@ -214,6 +214,9 @@ func makeVirtualServiceRoute(ctx context.Context, hosts sets.String, http *v1alp
 			Weight: 100,
 			Destination: &istiov1alpha3.Destination{
 				Host: privateGatewayServiceURLFromContext(ctx),
+				Port: &istiov1alpha3.PortSelector{
+					Number: 80,
```

Comment:
Is there a constant we can use?

Reply:
I dont think so unfortunately (but Id be happy to use one if you point can me at one!), I can create one but it'd be something like `const clusterGatewayHTTPPort = 80` so maybe not adding much?

Reply:
May be extract pkg/reconciler/ingress/resources/gateway.go#L314 and use the same?

Reply:
👍️ done.